### ISSUE TITLE
rds parameter group filter

### ISSUE THREAD

**Issue by kapilt** [capitalone/cloud-custodian#1054]
to verify various settings

**Comment by gfranxman** [comment]
Is this to filter by the group names or by family and type as well?

**Comment by gfranxman** [comment]
Digging through the api a little more, you mosts likely want something like

```yaml
resource: rds
filters:
  - parameter-group
     name: some-name-pattern
     status: some-apply-status   /* can be pending-reboot, in-sync */
```
where both params are optional, but one of the two must be present.

Is that right?

**Comment by kapilt** [comment]
implemented in #1248
